Cargo.toml

```diff
@@ -4,11 +4,16 @@ version = "0.1.0"
 authors = ["Luca Vaccaro <me@lvaccaro.com>"]
 edition = "2018"
 license = "MIT"
+description = "Bitcoin payment service, based on bdk and electrum."
+homepage = "https://github.com/lvaccaro/btctipserver"
```

Comment:
maybe `repository` instead of `homepage`?

Reply:
Or I've seen some crates use the same URL for both `homepage` and `repository`.

Reply:
update to repository for now.. maybe the project will be included in bdk a day